Makefile

```diff
@@ -71,6 +72,24 @@ clean:
 	$(MAKE) -C keymap clean
 
 distclean: clean
-	$(RM) -r orig root share
+	./extract.sh clean
 	./gzip.sh clean
 	./binutils.sh clean
+
+install: bin/123 share/man/man1/123.1
```

Comment:
Maybe the install rule should just depend on `all`?

Reply:
Agreed, made the change.